dp/generate_climos.py

```diff
@@ -85,9 +87,13 @@ def create_climo_files(outdir, input_file, t_start, t_end,
     """
     logger.info('Generating climo period %s to %s', d2s(t_start), d2s(t_end))
 
-    if input_file.is_multi_year_mean:
+    if input_file.is_multi_year:
```

Comment:
The exception raised below should either use a non-specific term like "climatologies" or specify the correct climatological operation, not always say "climatological means".

Reply:
Right, I should have caught that my mistake!